docs/package.json

```diff
@@ -15,8 +15,8 @@
     "docs"
   ],
   "scripts": {
-    "prepack": "node ./bin/copy-readmes && cd .. && npm run tsdocs",
```

Comment:
Just noticed that we have `copy-readmes`.. What does `copy-readmes` do? Why do we have it before?

Reply:
Fixed.